invokeai/frontend/web/src/features/controlNet/store/constants.ts

```diff
@@ -174,39 +174,98 @@ export const CONTROLNET_PROCESSORS = {
-  'lllyasviel/control_v11e_sd15_shuffle': 'content_shuffle_image_processor',
-  'lllyasviel/control_v11p_sd15_openpose': 'openpose_image_processor',
-  'CrucibleAI/ControlNetMediaPipeFace': 'mediapipe_face_processor',
+export const CONTROLNET_MODELS: Record<string, ControlNetModel> = {
+  'lllyasviel/control_v11p_sd15_canny': {
+    type: 'lllyasviel/control_v11p_sd15_canny',
+    label: 'Canny',
+    description: '',
+    defaultProcessor: 'canny_image_processor',
+  },
+  'lllyasviel/control_v11p_sd15_inpaint': {
+    type: 'lllyasviel/control_v11p_sd15_inpaint',
+    label: 'Inpaint',
+    description: 'Requires preprocessed control image',
+  },
+  'lllyasviel/control_v11p_sd15_mlsd': {
+    type: 'lllyasviel/control_v11p_sd15_mlsd',
+    label: 'M-LSD',
+    description: '',
+    defaultProcessor: 'mlsd_image_processor',
+  },
+  'lllyasviel/control_v11f1p_sd15_depth': {
+    type: 'lllyasviel/control_v11f1p_sd15_depth',
+    label: 'Depth',
+    description: '',
+    defaultProcessor: 'midas_depth_image_processor',
+  },
+  'lllyasviel/control_v11p_sd15_normalbae': {
+    type: 'lllyasviel/control_v11p_sd15_normalbae',
+    label: 'Normal Map (BAE)',
+    description: '',
+    defaultProcessor: 'normalbae_image_processor',
+  },
+  'lllyasviel/control_v11p_sd15_seg': {
+    type: 'lllyasviel/control_v11p_sd15_seg',
+    label: 'Segment Anything',
+    description: 'Requires preprocessed control image',
+  },
+  'lllyasviel/control_v11p_sd15_lineart': {
+    type: 'lllyasviel/control_v11p_sd15_lineart',
+    label: 'Lineart',
+    description: '',
+    defaultProcessor: 'lineart_image_processor',
+  },
+  'lllyasviel/control_v11p_sd15s2_lineart_anime': {
+    type: 'lllyasviel/control_v11p_sd15s2_lineart_anime',
+    label: 'Lineart Anime',
+    description: '',
+    defaultProcessor: 'lineart_anime_image_processor',
+  },
+  'lllyasviel/control_v11p_sd15_scribble': {
+    type: 'lllyasviel/control_v11p_sd15_scribble',
+    label: 'Scribble',
+    description: 'Requires preprocessed control image',
+  },
+  'lllyasviel/control_v11p_sd15_softedge': {
+    type: 'lllyasviel/control_v11p_sd15_softedge',
+    label: 'Soft Edge',
+    description: '',
+    defaultProcessor: 'hed_image_processor',
+  },
+  'lllyasviel/control_v11e_sd15_shuffle': {
+    type: 'lllyasviel/control_v11e_sd15_shuffle',
+    label: 'Content Shuffle',
+    description: '',
+    defaultProcessor: 'content_shuffle_image_processor',
+  },
+  'lllyasviel/control_v11p_sd15_openpose': {
+    type: 'lllyasviel/control_v11p_sd15_openpose',
+    label: 'Openpose',
+    description: '',
+    defaultProcessor: 'openpose_image_processor',
+  },
+  'lllyasviel/control_v11f1e_sd15_tile': {
+    type: 'lllyasviel/control_v11f1e_sd15_tile',
+    label: 'Tile (experimental)',
+  },
+  'lllyasviel/control_v11e_sd15_ip2p': {
+    type: 'lllyasviel/control_v11e_sd15_ip2p',
+    label: 'Pix2Pix (experimental)',
+    description: 'Requires preprocessed control image',
```

Comment:
Does not require preprocessed control image. Base images work directly. I feel saying needs preprocessed suggests the image needs to be processed in some way which is not particularly the case for these models.

Reply:
Ok - I will remove all descriptions for the moment as they are not exposed to the UI yet. Then somebody who knows what they all do can add proper descriptions